NOnion/Utility/HSUtility.fs

```diff
@@ -0,0 +1,74 @@
+﻿namespace NOnion.Utility
+
+open System
+
+open NOnion
+open DateTimeUtils
+
+module HSUtility =
```

Comment:
lets avoid HS acronym

Reply:
Fixed in [aae9fe3](https://github.com/nblockchain/NOnion/pull/30/commits/aae9fe32759d73c848bad5769597077cf7026598)